fdbserver/VersionedBTree.actor.cpp

```diff
@@ -3744,7 +3835,10 @@ class DWALPager final : public IPager2 {
 #pragma pack(push, 1)
 	// Header is the format of page 0 of the database
 	struct Header {
-		static constexpr int FORMAT_VERSION = 8;
+		static constexpr int FORMAT_VERSION = 10;
```

Comment:
nit: seems like 9 is missing :D

Reply:
Haha, yeah it's probably in the edit history of this PR.  I increment whenever I think I broke something.